CMakeLists.txt

```diff
@@ -79,9 +79,9 @@ if(FAISS_ENABLE_GPU)
   endif()
 endif()
 
-if(FAISS_ENABLE_RAFT AND NOT TARGET raft::raft)
-  find_package(raft COMPONENTS compiled distributed)
```

Comment:
Is RAFT still needed for compilation or we need to install rmm explicitly? 

https://github.com/facebookresearch/faiss/actions/runs/10889999572/job/30218385769?pr=3549
```
The link interface of target "cuvs::cuvs" contains:

    rmm::rmm

  but the target was not found.  Possible reasons include:
  ```

Reply:
@asadoughi I tried linking `raft::raft` but now I see another failure in ci https://github.com/facebookresearch/faiss/actions/runs/10908718713/job/30275337629?pr=3549

Reply:
rmm and raft should ideally be available through cuvs. But I have also tried explicitly linking `raft::raft`.